src/tinyformat.h

```diff
@@ -1126,6 +1131,7 @@ template<TINYFORMAT_ARGTYPES(n)>                                          \
 std::string format(const char* fmt, TINYFORMAT_VARARGS(n))                \
 {                                                                         \
     std::ostringstream oss;                                               \
+    oss.imbue(std::locale::classic()); /* Added for Bitcoin Core */       \
```

Comment:
Have you tried submitting this upstream? Other users might be interested in this, but it might have to be hidden behind a compile time flag.

Reply:
My plan is to do that if/when we get this in to Core :)